hangout_bot/constant.py

```diff
@@ -21,20 +21,26 @@
 LAST_DAY_MONTH = 31
 
 # PHASE_CONSTANTS (order of configuration)
+DELETE_CAMPAIGN = -3
+VIEWING_CAMPAIGNS = -2
 INACTIVE = -1
-NAME = 0
-START_DATE = 1
-END_DATE = 2
-DAILY_BUDGET = 3
-MANUAL_CPC = 4
-DOMAIN = 5
-TARGETS = 6
-LOCATIONS = 7
-NEG_LOCATIONS = 8
-AD_TEXT = 9
-SUBMISSION = 10
+KEYWORD_CAMPAIGN = 0
+NAME = KEYWORD_CAMPAIGN + 1
+START_DATE = NAME + 1
+END_DATE = START_DATE + 1
+DAILY_BUDGET = END_DATE + 1
+MANUAL_CPC = DAILY_BUDGET + 1
+DOMAIN = MANUAL_CPC + 1
+TARGETS = DOMAIN + 1
+LOCATIONS = TARGETS + 1
+NEG_LOCATIONS = LOCATIONS + 1
+AD_TEXT = NEG_LOCATIONS + 1
+SUBMISSION = AD_TEXT + 1
```

Comment:
Have you considered using an enum for these? Would likely be easier than this, as this isn't intuitive.

Reply:
Added